.github/workflows/mypy_primer_comment.yml

```
@@ -110,7 +110,7 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body,
+              body: body.substr(0, 60000),
```

This is huge. If a line is 100 characters long on average, then this allows 600 lines of mypy_primer output. How about allowing only about 300 lines?

```suggestion
              body: body.substr(0, 30000),  // About 300 lines
```